p256-crypto/src/multi_signature.rs

```diff
@@ -94,6 +98,7 @@ impl TryFrom<MultiSignature> for Signature {
 
 #[derive(Eq, PartialEq, Ord, PartialOrd, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+// SBP-M1 review: same as previous comment
 pub enum MultiSigner {
     /// An Ed25519 identity.
     Ed25519(ed25519::Public),
```

Comment:
Same thing here, the corresponding type in substrate can be found here:

https://github.com/paritytech/substrate/blob/1837f423b494254e1d27834b1c9da34b2c0c2375/primitives/runtime/src/lib.rs#L303